tests/integration/create_token.py

```diff
@@ -51,11 +92,48 @@ def _get_payload_with_params(schema_name, schema_url=None, **extra_payload):
 
         return payload_vars
 
+    @staticmethod
+    def _get_payload_with_params_v2(schema_name, payload, **extra_payload):
```

Comment:
What different about this than the `_get_payload_with_params` above?

Reply:
I've refactored this now 👍